<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>371</width>
      <height>471</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background: rgb(144, 255, 209)</string>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="page_5">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>131</width>
        <height>411</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-241</y>
         <width>112</width>
         <height>650</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_3">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_4">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_22">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_21">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_20">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_19">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_18">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_17">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>40</y>
        <width>231</width>
        <height>401</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="page">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>297</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 1</string>
       </attribute>
       <widget class="QPushButton" name="pushButton_23">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>50</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>231</width>
         <height>297</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Page 2</string>
       </attribute>
       <widget class="QToolButton" name="toolButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>70</y>
          <width>37</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <attribute name="label">
        <string>页</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>20</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>70</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <attribute name="label">
        <string>页</string>
       </attribute>
       <widget class="QCheckBox" name="checkBox">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>110</y>
          <width>71</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>CheckBox</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>301</width>
        <height>411</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>2</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>100</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>150</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <widget class="QRadioButton" name="radioButton_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>130</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButton_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>190</y>
          <width>89</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>RadioButton</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>页</string>
       </attribute>
       <widget class="QCommandLinkButton" name="commandLinkButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>140</y>
          <width>172</width>
          <height>41</height>
         </rect>
        </property>
        <property name="text">
         <string>CommandLinkButton</string>
        </property>
       </widget>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>230</y>
          <width>156</width>
          <height>23</height>
         </rect>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnChange">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>251</width>
      <height>271</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
